qiskit/backends/_qeremote.py

```diff
@@ -0,0 +1,112 @@
+import time
+import logging
+from qiskit.backends._basebackend import BaseBackend
+from qiskit.backends._backendutils import get_backend_configuration
+from qiskit import _openquantumcompiler as openquantumcompiler
+from qiskit._result import Result
+from IBMQuantumExperience.IBMQuantumExperience import IBMQuantumExperience
+
+logger = logging.getLogger(__name__)
+
+class QeRemote(BaseBackend):
+    def __init__(self, configuration=None):
+        """Initialize remote backend for IBM Quantum Experience.
+
+        Args:
+
+        """
+        if configuration is None:
+            self._configuration = get_backend_configuration(backend_name)
+        else:
+            self._configuration = configuration
+        self._configuration['local'] = False
+
+    def run(self, q_job):
+        """Run jobs
+
+        Args:
+            q_job (QuantumJob): job to run
+
+        Returns:
+            Result object.
+
+        Raises:
+            ResultError: if the api put 'error' in its output
+        """
+        self._qobj = q_job.qobj
```

Comment:
For the backends that do not require `_qobj` to be available after `run()`, would it be possible to _not_ assign it to an instance variable, just using `qobj = q_job.qobj` (or any other name or direct usage that you see fit)? I haven't checked if this is possible in all the backends, but it would fit the principle of avoiding functions to have side effects such as adding instance attributes inadvertently, and make the code slightly more readable.

For those cases where `self._qobj` is needed outside `run()` or has a longer lifetime, initializing it in `__init__` (with `None` or similar) would also be recommended.

Reply:
Yes, I'll remove assignment.